content/concepts/accounts/decentralized-identifiers.md

```diff
@@ -0,0 +1,82 @@
+---
+html: decentralized-identifiers.html
+parent: accounts.html
+blurb: Decentralized identifiers enable verifiable, decentralized digital identities.
+status: not_enabled
+labels:
+  - DID
+---
+# Decentralized Identifiers
+
+_(Requires the [DID amendment][] :not_enabled:)_
+
+A Decentralized Identifier (DID) is a new type of identifier defined by the World Wide Web Consortium (W3C) that enables verifiable, digital identities. DIDs are fully under the control of the DID owner, independent from any centralized registry, identity provider, or certificate authority.
+
+The key principles of a DID are:
+
+- **Decentralization:** No central issuing agency controls the DID, enabling the owner to update, resolve, or deactivate it.
+
+- **Cryptographically Verifiable:** DIDs are verified through cryptographic proofs, making them tamper-evident and secure.
+
+- **Interoperability:** DIDs are open to any solution that recognizes the W3C DID standard. This means a DID can be used to authenticate and establish trust in various digital transactions and interactions.
+
+**Note:** The implementation of DIDs on the XRP Ledger conforms to the requirements in the [DID v1.0 specification](https://www.w3.org/TR/did-core/).
+
+
+## How It Works
+
+1. An XRPL account holder generates a DID that is controlled by the account.
+2. The DID is associated with a DID document as defined by W3C specifications.
+3. The DID is used for digital tasks such as:
+    - Signing digital documents.
+    - Making secure online transactions.
+    - Logging into websites.
+4. The verifier resolves the DID to its document to verify the subject's identity.
+
+
+## DID Documents
+
+DID documents contain the necessary information to cryptographically verify the identity of the subject described by a DID document. The subject can be a person, organization, or thing. For example, a DID document could contain cryptographic public keys that the DID subject can use to authenticate itself and prove its association with the DID.
+
+**Note:** DID documents usually serialize to a JSON or JSON-LD representation.
+
+On the XRP Ledger, there are several ways to associate a DID to a DID document:
+
+1. Store a reference to the document in the `URI` field of the `DID` object, which points to a document stored on another decentralized storage network, such as IPFS or STORJ.
+2. Store a minimal DID document in the `DIDDocument` field of the `DID` object.
+3. Specify a minimal _implicit_ DID document generated from the DID and other available public information.
```

Comment:
Super-nit: it feels weird to me to "specify" an "implicit" document. Maybe this wording would be better:

```suggestion
3. Use a minimal, _implicit_ DID document generated from the DID and other available public information.
```